engine/modelc/src/modelimporter_gltf.cpp

```diff
@@ -1328,21 +1350,38 @@ static void LoadChannel(NodeAnimation* node_animation, cgltf_animation_channel*
     cgltf_accessor* accessor_times = channel->sampler->input;
     cgltf_accessor* accessor = channel->sampler->output;
 
+    uint32_t key_count = accessor_times->count;
+
+    // 1 for 1xf32, 4 for 4xf32
     uint32_t num_components = (uint32_t)cgltf_num_components(accessor->type);
+    // number of items per time step
+    uint32_t num_items = accessor->count / key_count;
```

Comment:
For morph targets (e.g. 2 of them), the accessor count is larger than the times count, and "num_components == 1"
For regular skeleton animation data, they are the same count, but the "num_components == 3" (or 4)

Still unverified fix in terms of correctness, but at least it's not crashing.
We'll get to this when supporting morph targets.